modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -1594,13 +1593,20 @@ private void _createTable(
 			}
 			else if (objectField.hasUniqueValues()) {
 				indexable = true;
-				unique = true;
 			}
 
 			if (!indexable) {
 				continue;
 			}
 
+			boolean unique = false;
+
+			if (objectField.compareBusinessType(
+					ObjectFieldConstants.BUSINESS_TYPE_AUTO_INCREMENT)) {
```

Comment:
Hey @pedro-oliveira446 , as we discussed, let's remove the database constraint for auto increment field also.

Reply:
Done!